src/wallet/rpcdump.cpp

```diff
@@ -679,7 +710,13 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
             } else {
                 file << "change=1";
```

Comment:
when export hd sigma this one should printed valid value or should not be shown at all.

Reply:
@thebevrishot Could you clarify what you mean by this? Do you mean that if it is a mint it should show `change=2`?

Reply:
sorry, I'm confused this line with nChange in hdkeypath.

Reply:
Ah sure, ok.